### ISSUE TITLE
[Request] Update Thaumcraft support to TC6

### ISSUE THREAD

**Issue by tgapira** [jaquadro/StorageDrawers#660]
In the new iteration of Thaumcraft, drawers do not display the aspect symbol of contained Vis Crystals or Essentia Phials, which would be a tremendous help with ease of use.

**Comment by Graywaren** [comment]
Yes please. I was super excited to see this compat on the page and so disappointed when it didn't work.
